classes/mail/variables/SubmissionEmailVariable.php

```diff
@@ -79,7 +79,7 @@ public function values(string $locale): array
             self::SUBMISSION_ABSTRACT => PKPString::stripUnsafeHtml($this->currentPublication->getLocalizedData('abstract', $locale)),
             self::SUBMISSION_ID => (string) $this->submission->getId(),
             self::SUBMISSION_PUBLISHED_URL => $this->getSubmissionPublishedUrl($this->getContext()),
-            self::SUBMISSION_TITLE => htmlspecialchars($this->currentPublication->getLocalizedFullTitle($locale)),
+            self::SUBMISSION_TITLE => PKPString::stripUnsafeHtml($this->currentPublication->getLocalizedFullTitle($locale, 'html'), 'allowed_title_html'),
```

Comment:
If you find you have to repeat this pattern very often, another option would be to move the HTMLPurifier invocation *inside* `getLocalizedFullTitle`, to perform the filtering there. I'm ambivalent about the idea.

Reply:
there is very few such use of this pattern as right now I ma using a sanitization process before store the title and sub title . And now that you have mentioned it, I think it can be removed if we keep the sanitization process .